cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -607,13 +601,14 @@ def run_call_back(self, status_kw_name='action') -> typing.Tuple[str, typing.Uni
                     logger=self.logger,
                     decoded_token=self.decoded_token,
                     job_id=self.job_id,
+                    session_id=self.par_dic['session_id'],
                     status=status,
                     instrument=instrument_name,
                     product_type=product_type,
                     time_request=time_original_request,
                     request_url=products_url,
                     api_code=DispatcherAPI.set_api_code(original_request_par_dic, 
-                                                        url=self.app.config['conf'].products_url),
+                                                        url=self.app.config['conf'].products_url + "/dispatch-data"),
```

Comment:
Will it always be the case to have `dispatch_data` as final part of the url ? I guess this depends on the deployment...

Reply:
It does not, it is how frontend-astrooda refers to the dispatcher, it's also hardcoded in all instruments.
e.g. there:
https://github.com/oda-hub/frontend-astrooda/blob/master/instruments/astrooda_isgri/astrooda_isgri.inc#L22

since it is a single unique alias, there is no reason for it to be configurable.

products_url is frontend, where users click. dispatch-data makes it dispatcher.
I considered to add a comment there since it's not obvious. I will do it.